play/apps/tournament/models.py

```diff
@@ -29,29 +29,42 @@ class Meta:
 class Tournament(models.Model):
     name = models.CharField(max_length=256)
 
-    def create(self):
-        tournament = Tournament.objects.create()
+    def build_structure(self):
+        numSnakes = Snake.objects.all().count()
+        while numSnakes < 40:
+            snake = random.choice(list(Snake.objects.all()))
```

Comment:
does this run a query against the db each time?

Reply:
ooh, this is bad for me to have left in. It was my scratch code to create 40 snakes so I could check what the tournament structure would look like a little more realistically. I need to remove those lines